packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/BarChartViewer/BarChartViewerContainer.spec.js

```diff
@@ -0,0 +1,189 @@
+import { shallow } from 'enzyme'
+import React from 'react'
+import nock from 'nock'
+import sinon from 'sinon'
+
+import { BarChartViewerContainer } from './BarChartViewerContainer'
+import {
+  variableStarAmplitudeMockData,
+  variableStarPeriodMockData
+} from './mockData'
+import { Factory } from 'rosie'
+
+const subject = Factory.build('subject', {
+  locations: [
+    { 'application/json': 'http://localhost:8080/mockData.json' }
+  ]
+})
+
+const nextSubject = Factory.build('subject', {
+  locations: [
+    { 'application/json': 'http://localhost:8080/nextSubject.json' }
+  ]
+})
+
+const subjectJSON = variableStarAmplitudeMockData
+const nextSubjectJSON = variableStarPeriodMockData
+
+const imageSubject = Factory.build('subject')
+
+const failSubject = Factory.build('subject', {
+  locations: [
+    { 'application/json': 'http://localhost:8080/failure.json' }
+  ]
+})
+
+describe('Component > BarChartViewerContainer', function () {
+  it('should render without crashing', function () {
+    const wrapper = shallow(<BarChartViewerContainer />)
+    expect(wrapper).to.be.ok()
+  })
+
+  it('should mount with an initialized state', function () {
```

Comment:
`shallow` started to support lifecycle methods starting v3: https://airbnb.io/enzyme/docs/api/shallow.html

However, good point on the description of this spec. I'll reword that since I'm not actually doing a full mount here.